support/openharmony/build-profile.json5

```diff
@@ -4,16 +4,16 @@
       {
         "name": "default",
         "signingConfig": "default",
-        "compileSdkVersion": 12,
-        "compatibleSdkVersion": 12,
-        "targetSdkVersion": 12,
+        "compileSdkVersion": 15,
+        "compatibleSdkVersion": 15,
+        "targetSdkVersion": 15,
         "runtimeOS": "OpenHarmony"
```

Comment:
I would prefer if we keep CI at our minimum supported OH version. The OH developer phone still only has API-12 so increasing the minimum version in CI, would mean that we could accidentally break support for older versions at any time.